From 50a6b6e40557352e68e918da0cccecfc85047284 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Tue, 2 Feb 2016 09:50:35 +0100
Subject: [PATCH 11/12] S Core: Add support for USB Interfaces

Configure the two USB Interface. USB1 is configured as master and USB2
is configured as peripheral.

Ticket #872
---
 arch/arm/boot/dts/imx6sx-score.dts | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-score.dts b/arch/arm/boot/dts/imx6sx-score.dts
index f7b8e06..9325bc7 100644
--- a/arch/arm/boot/dts/imx6sx-score.dts
+++ b/arch/arm/boot/dts/imx6sx-score.dts
@@ -83,6 +83,16 @@
     status = "okay";
 };
 
+&usbotg1 {
+	dr_mode = "host";
+	status = "okay";
+};
+
+&usbotg2 {
+	dr_mode = "peripheral";
+	status = "okay";
+};
+
 &usdhc3 {
     pinctrl-names = "default";
     pinctrl-0 = <&pinctrl_usdhc3>;
-- 
1.9.1

